src/nodes/display/PixelationNode.js

```diff
@@ -0,0 +1,239 @@
+import TempNode from '../core/TempNode.js';
+import { uv } from '../accessors/UVNode.js';
+import { addNodeElement, tslFn, nodeObject, vec2, vec3, float, If } from '../shadernode/ShaderNode.js';
+import { NodeUpdateType } from '../core/constants.js';
+import { uniform } from '../core/UniformNode.js';
+import { dot, clamp, smoothstep, sign, step, floor } from '../math/MathNode.js';
+import { Vector4 } from '../../math/Vector4.js';
+import { property } from '../core/PropertyNode.js';
+import QuadMesh from '../../renderers/common/QuadMesh.js';
+import { RenderTarget } from '../../core/RenderTarget.js';
+import { passTexture } from './PassNode.js';
+
+const createEdgesQuad = new QuadMesh();
+const lowerResolutionQuad = new QuadMesh();
+
+class PixelationNode extends TempNode {
+
+	constructor( textureNode, depthNode, normalNode, pixelSizeNode, normalEdgeStrength, depthEdgeStrength ) {
+
+		super();
+
+		// Input textures
+
+		this.textureNode = textureNode;
+		this.depthNode = depthNode;
+		this.normalNode = normalNode;
+
+		// Input uniforms
+
+		this.pixelSizeNode = pixelSizeNode;
+		this.normalEdgeStrength = normalEdgeStrength;
+		this.depthEdgeStrength = depthEdgeStrength;
+
+		// Private uniforms
+
+		this._resolution = uniform( new Vector4() );
+
+		// Intermediary render targets
+
+		this._createEdgesRT = new RenderTarget();
+		this._createEdgesRT.texture.name = 'PixelationNode.renderEdges';
+		this._lowerResolutionRT = new RenderTarget();
+		this._lowerResolutionRT.texture.name = 'PixelationNode.lowerResolution';
```

Comment:
We should get closer to the original look if you configure both render targets with nearest filtering like so:
````js
this._createEdgesRT = new RenderTarget();
this._createEdgesRT.texture.name = 'PixelationNode.renderEdges';
this._createEdgesRT.texture.minFilter = NearestFilter;
this._createEdgesRT.texture.magFilter = NearestFilter;
this._lowerResolutionRT = new RenderTarget();
this._lowerResolutionRT.texture.name = 'PixelationNode.lowerResolution';
this._lowerResolutionRT.texture.minFilter = NearestFilter;
this._lowerResolutionRT.texture.magFilter = NearestFilter;
```
Right now, linear filtering is used which is responsible for the additional blurred edges.

Reply:
Thank you, can't believe I forgot that.